pipeline-templates/job/code-build.yml

```diff
@@ -1,47 +1,43 @@
-parameters:
-  SolutionBaseName:
-  BuildConfiguration:
-
+  
 jobs:
 - job: CodeBuild
+  pool:
+    name: DAS - Continuous Integration
   variables:
-  - group: BUILD Management Resources
+  - group: Prod Management Resources

```

Comment:
this should be `BUILD Management Resources`

Reply:
Whoops, that's updated 👍